src/camlsnark_c/libsnark-caml/depends/xbyak/test/misc.cpp

```diff
@@ -1,9 +1,9 @@
 #include <stdio.h>
 #include <string.h>
 #include <string>
-#include <xbyak/xbyak.h>
-#include <cybozu/inttype.hpp>
-#include <cybozu/test.hpp>
+#include "xbyak/xbyak.h"
```

Comment:
Opposed to these, see [this comment](https://github.com/o1-labs/snarky/pull/524#issuecomment-681968731).